include/aspect/postprocess/composition_velocity_statistics.h

```diff
@@ -66,7 +66,7 @@ namespace aspect
          */
 
       private:
-        std::vector<std::string> slab_compositions;
+        std::vector<std::string> selected_field_compositions;
```

Comment:
This name is a bit confusing. Several 'compositional fields' can make up an entity of which the rms velocity is to be computed. Maybe call it 'selected_fields' or 'selected_compositional_fields'?